### ISSUE TITLE
xSQLServerAlwaysOnAvailabilityGroup: Reports wrongly that 'NT AUTHORITY\SYSTEM' is missing even if it is not

### ISSUE THREAD

**Issue by johlju** [PowerShell/xSQLServer#467]
**Details of the scenario you try and problem that is occurring:**
When 'NT SERVICE\ClusSvc' exist as a login with the correct permission, the resource reports that 'NT AUTHORITY\SYSTEM' is missing even if it is not.

```
VERBOSE: 2017-04-02_12-13-34: The login 'NT AUTHORITY\SYSTEM' is not present.
```

It is because when the corrrect permission is found for the 'NT SERVICE\ClusSvc' then on the next loop (next login to check) the `$clusterPermissionsPresent` is no longer $false and the wrong verbose message is outputed.
```
if ( $serverObject.Logins[$loginName] -and -not $clusterPermissionsPresent )
```

Instead it should stop checking for permission as soon as the best recommended account has the correct permission.

**The DSC configuration that is using the resource (as detailed as possible):**
```
xSQLServerAlwaysOnAvailabilityGroup $resourceConfigName
{
    Ensure = 'Present'
    Name = $Node.AvailabilityGroupName
    SQLInstanceName = $Node."$($currentSqlInstance)InstanceName"
    SQLServer = $Node.NodeName

    PsDscRunAsCredential = $SqlAdministratorCredential

    DependsOn = "[xSQLServerEndpoint]$($currentSqlInstance)-HADREndpoint",
                "[xSQLServerPermission]$($currentSqlInstance)-AddNTServiceClusSvcPermissions"
}
```

Or with both account present in SQL Server and running this 
```
Set-TargetResource -SQLServer $env:COMPUTERNAME -SQLInstanceName 'SQL2016' -Name 'TESTCLU02A' -Ensure Present 
```

**Version of the Operating System, SQL Server and PowerShell the DSC Target Node is running:**
Windows Server 2012, SQL Server 2016 and WMF 5.0

**Version of the DSC module you're using, or 'dev' if you're using current dev branch:**
Dev

**Comment by randomnote1** [comment]
@johlju , I see what happened here. I'll get this fixed up.

**Comment by johlju** [comment]
Resolved in merged PR #470
